packages/vendor-core/package.json

```diff
@@ -34,7 +34,7 @@
     "classnames": "^2.2.5",
     "datatables.net": "~1.10.12",
     "datatables.net-bs": "~1.10.12",
-    "diff": "~3.0.0",
```

Comment:
@glekner, @ohadlevy, what is the reason we locked on `~3.0.0`?
There was a specific problem with `3.1.0` or it was just a policy?